roles/vicepresidentinternal.rst

```diff
@@ -0,0 +1,87 @@
+Vice President Internal
+=======================
+
+Roles and Responsibilities
+--------------------------
++ **Organize** -
+  The Vice President Internal (VP-I) must maintain that operations are running
+  efficiently.
+  Take thorough notes at the Executive meetings and make them available to all
+  members of the board.
+  Try to find ways to make the ship sail smoother.
+  The roles of the VP-I work to this end.
++ **Communicate** -
+  The VP-I must be the point of contact for committees.
+  Keep track of deadlines, membership, and keep a model of each committee's
+  power structure. (Chair, Vice-Chair, etc.)
+  Keep the right people in the loop with new information.
+  This will give you the information necessary to be useful to the rest of your
+  executive board.
++ **Comply** -
+  Since S&T ACM lacks a parliamentarian role, the VP-I acts as such.
+  This means overseeing the meetings in both the parliamentarian position and in
+  a Vice President capacity.
+  Contribute ideas and thoughts, as well as maintain order.
+  Make sure all comments are heard and that no one's comments are suppressed by
+  other members of the board.
+  Be familiar with the constitution and follow it as strictly as possible.
+  This will ensure that no one is treated unfairly and poor decisions are less
+  often to occur.
+  This also means that the VP-I's job to make sure that S&T ACM's branding never
+  breaks university policy, and that the constitution is up to date and
+  compliant with regulation.
++ **Manage Elections** -
+  The VP-I is in charge of all Election details.
+  This includes announcments, scheduling, vote proceedings, and counting votes.
+  Do so ethically, as you will be checked.
++ **Plan Events** -
```

Comment:
I changed this so the VPE is in charge of all events. It may be better to say something like `The VPI should ensure the VPE has all the help they need to handle events`.